drivers/scheduler/k8s/k8s.go

```diff
@@ -591,6 +593,24 @@ func validateSpec(in interface{}) (interface{}, error) {
 	return nil, fmt.Errorf("unsupported object: %v", reflect.TypeOf(in))
 }
 
+// relatedVolume will test if the given volume should be included. Currently that just relates to
+// if we're filtering by Pure volumes, but can be expanded later.
+func (k *K8s) relatedVolume(claim *v1.PersistentVolumeClaim) (bool, error) {
```

Comment:
What does 'related' mean? It does not convey the intention clearly.

Reply:
Agreed, I was struggling to think of a good term at the time. Will revise